ansible/idr-playbooks/group_vars/omero-hosts.yml

```diff
@@ -77,6 +77,110 @@ omero_web_app_add_top_links:
     target: new
     title: Open OMERO user guide in a new tab
 
+# Metadata Panes order
+omero_web_ui_metadata_panes:
+- name: "map"
+  label: "Attributes"
+  index: 0
+- name: "table"
+  label: "Tables"
+  index: 1
+- name: "file"
+  label": "Attachments"
+  index: 2
+- name: "comment"
+  label: "Comments"
+  index: 3
+- name: "tag"
+  label: "Tags"
+  index: 4
+- name: "rating"
+  label: "Ratings"
+  index: 5
+- name: "other"
+  label: "Others"
+  index: 6
+
+# Mapr config
+omero_web_mapr_config:
+- menu: "gene"
+  label: "Gene"
+  config:
+    default:
+      - "Gene Symbol"
+    all:
+      - "Gene Symbol"
+      - "Gene Identifier"
+    ns:
+      - "openmicroscopy.org/mapr/gene"
+- menu: "genesupplementary"
+  label: "Gene supplementary"
+  config:
+    default:
+    all:
+    ns:
+      - "openmicroscopy.org/mapr/gene/supplementary"
+- menu: "sirna"
+  label: "siRNA"
+  config:
+    default:
+      - "siRNA Identifier"
+    all:
+      - "siRNA Identifier"
+      - "siRNA Pool Identifier"
+    ns:
+      - "openmicroscopy.org/mapr/sirna"
+- menu: "sirnasupplementary"
+  label: "siRNA supplementary"
+  config:
```

Comment:
this should be translated to `config: []`, if the key is missing we have find a different way.